docs/adr/0010_release_strategy.md

```diff
@@ -0,0 +1,247 @@
+# ADR: Monorepo Versioning, Branching & Release Strategy
+
+* **Status:** proposed
+* **Deciders:** OCM Technical Steering Committee
+* **Date:** 2025‑09-30
+
+---
+
+## Technical Story
+
+Enable fast, reliable releases from the new monorepo using Git‑native, tag‑only versioning and a simple branching model, while keeping a clean path to introduce an optional root component later.
+
+### Context and Problem Statement
+
+We have already consolidated two components into our monorepo
+
+* cli - CLI v2
+* kubernetes/controller - controller v2
+
+and will add a third component later:
+
+* website - OCM website
+
+The team already operates GitHub‑based release workflows for OCM v1. We considered carrying over VERSION files, but decided to converge on **Git‑native, tag‑only** versioning for a single, immutable source of truth.
+
+### Important scope decisions
+
+* **Lockstep MAJOR/MINOR:** Both sub‑components finalize **the same X.Y** at sprint start. This is supportive for an `ocm` root component to be introduced later, which will share X.Y and patch in tandem when any sub-component patches.
+* **Per‑component PATCH:** Patch releases may happen independently per sub‑component **between** sprints.
+* **Promotion of RC to final** happens using the same release artifacts (no rebuild).
+
+### Out of scope (for this ADR)
+
+* **Root ocm component** Not implemented in this ADR; later, it will share **X.Y** and **patch in tandem** when any sub‑component patches.
```

Comment:
```suggestion
* **Root ocm component** Not implemented in this ADR; later, it will share **X.Y** and **patch in tandem** when any sub‑component patches are created.
```